### ISSUE TITLE
check_mk 2.1.0 support

### ISSUE THREAD

**Issue by syncopsta** [HeinleinSupport/check_mk_extensions#109]
By updating to Checkmk version 2.1.0, both the dew point sensor and the temperature sensor throw errors.
These are caused by changes in check_temperature, because 'unique_name' and 'value_store' must be set for a trend computation.
See also: https://checkmk.com/werk/13730

**Comment by jplitza** [comment]
Also:
```
ImportError: cannot import name 'perfometer_linear' from 'cmk.gui.plugins.views.perfometers' (unknown location)                                                                                                                                                                  
```
because that import apparently moved to `cmk.gui.plugins.views.perfometers.check_mk`

**Comment by gurubert** [comment]
fixed with commit b8b91bd6968044041a9b20b588210676939398b1
